subsys/bluetooth/Kconfig

```diff
@@ -222,6 +222,64 @@ config BT_DATA_LEN_UPDATE
 	  Enable support for Bluetooth v4.2 LE Data Length Update procedure.
 endif # BT_CONN
 
+
+config BT_ISO
+	bool "Bluetooth Isochronous Channel Support [EXPERIMENTAL]"
+	help
+	  Select this to enable Isochronous Channel support.
+
+if BT_ISO
+
+config BT_ISO_MAX_CONN
+	int "Maximum number of simultaneous ISO connections"
+	depends on BT_ISO
+	default BT_MAX_CONN
+	range 1 64
+	help
+	  Maximum number of simultaneous Bluetooth isochronous connections
+	  supported.
+
+config BT_ISO_TX_BUF_COUNT
+	int "Numer of Isochronous TX buffers"
```

Comment:
What "Isochronous TX buffers" is this referring to? HCI ISO Data packets? SDUs? PDUs? The terminology may have been clear on the host side, but it is in my opinion too fuzzy when being shared with the controller.

(also: Numer->Number)

Reply:
I fixed the numer->number part. For the other part, I agree that it is a bit fuzzy, and probably not something I can answer fully. My understanding of hci_raw.c is that it's partially host and partially controller. As far as I can tell, it's being used in the host for ISO Data packets (either over HCI or proprietary), but also for HCI in setups using raw_hci or rpmsg. 

@Vudentz  @jhedberg anything you'd like to add here?

Reply:
I guess it quite obvious it is for ISO data packets as SDU, etc, are in the controller side but perhaps this will get more confusing if we put this as common Kconfig.